groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group13/AluminiumChain.groovy

```diff
@@ -223,14 +223,30 @@ BR.recipeBuilder()
     .EUt(200)
     .buildAndRegister()
 
-// Ullman's - Aluminium Ch. 5. Hall-Heroult makes like 99.9% pure already so not making it a chanced output.
-ZONE_REFINER.recipeBuilder()
-    .inputs(ore('ingotAluminium'))
-    .outputs(metaitem('ingotHighPurityAluminium'))
-    .duration(200)
+// High purity aluminium (Ullman's - Aluminium Ch. 5)
+
+BR.recipeBuilder()
+    .inputs(ore('dustBariumCarbonate') * 5)
+    .fluidInputs(fluid('ammonium_fluoride_solution') * 2000)
+    .outputs(metaitem('dustBariumFluoride') * 3)
+    .fluidOutputs(fluid('ammonium_carbonate_solution') * 1000)
+    .duration(100)
     .EUt(VA[LV])
     .buildAndRegister()
 
+ELECTROLYTIC_CELL.recipeBuilder()
+    .notConsumable(metaitem('graphite_electrode'))
+    .notConsumable(fluid('sodium_fluoride') * 576)
```

Comment:
nvm maybe we should mix the solid first together as a new material and then melt afterwards.